.travis.yml

```diff
@@ -21,6 +22,11 @@ before_install:
     - travis_retry sudo apt-get -y -qq update
     # get Octave 4,0
     - travis_retry sudo apt-get -y -qq install octave
```

Comment:
Yes: shippable.com can use .travis.yml files; I think it translates them, and in the process replaces travis_retry by some own function.